packages/http-client-csharp/emitter/src/lib/converter.ts

```diff
@@ -172,6 +172,7 @@ export function fromSdkModelType(
             ? flattenedNamePrefixes.concat(property.name)
             : undefined,
         Decorators: property.decorators,
+        InputModel: inputModelType!,
```

Comment:
I would have expected to see Input type class updates in MGC.Input and some deserialization code updates so its read in property?